### PR TITLE
Fixing kubevirt config path argument

### PR DESCRIPTION
**Pull request by pkliczewski**
We expected to get location of kubernetes config file provided as env
variable and there was no way to configure it only in config file. This
PR moves this setting to config file.

Fixes: https://bugzilla.redhat.com/1654652
Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
@wottop please review

**Comment by pkliczewski** [comment]
@wottop can you please take a look?

**Comment by wottop** [comment]
We are working to remove environment variables from the virt-who process. Is there a way to use the virt-who.conf file it achieve this functionality?

**Comment by pkliczewski** [comment]
@wottop This patch enables usage of config file. Before this change we were dependent on environment variable.

File: virt-who.conf [hunk 1]
```diff
@@ -90,5 +90,5 @@ VIRTWHO_DEBUG=0
 # Options for Kubevirt mode
 #VIRTWHO_KUBEVIRT_OWNER=
 #VIRTWHO_KUBEVIRT_ENV=
-#KUBECONFIG=
+#VIRTWHO_KUBEVIRT_KUBECONFIG=
```
Comment (wottop) [inline virt-who.conf:93]: This is the name of an environment variable. Shouldn't it be 'kubeconfig'?

**Review by wottop — CHANGES_REQUESTED** [review]

File: virt-who.conf (hunk 1 above)
Comment (pkliczewski) [reply virt-who.conf:93]: We are not using environment variable anymore. We rely on command line or config file now.

Comment (wottop) [reply virt-who.conf:93]: then remove this line from this file.

Comment (wottop) [reply virt-who.conf:93]: Are you aware that you can specify a single config file on the command line, using '-c'?

Comment (pkliczewski) [reply virt-who.conf:93]: I did not know this option. Thanks for a hint.

Comment (wottop) [reply virt-who.conf:93]: Does that reduce the necessity of this change?

Comment (pkliczewski) [reply virt-who.conf:93]: No, currently kubevirt plugin won't use this property from config file at all. This PR makes it possible.

**Comment by wottop** [comment]
ack

**Review by wottop — APPROVED** [review]

**Comment by pkliczewski** [comment]
:tada:

**Pull request merged**